consensus/src/round_manager.rs

```diff
@@ -78,6 +78,7 @@ use fail::fail_point;
 use futures::{channel::oneshot, stream::FuturesUnordered, Future, FutureExt, SinkExt, StreamExt};
 use lru::LruCache;
 use serde::Serialize;
+use sha3::digest::generic_array::typenum::private::IsEqualPrivate;
```

Comment:
what is this?

Reply:
Lol some autoimport mishap. Should fail lint.